IaC/pmm.cd/init.groovy.d/matrix.groovy

```diff
@@ -152,6 +152,7 @@ authz_strategy_config = [
         'atymchuk': ['Overall Administer'],
         'talhabinrizwan': ['Overall Administer'],
         'puneet0191' : ['Overall Read','Credentials Create','Credentials Update','Credentials View','Agent Build','Agent Configure','Agent Connect','Agent Create','Agent Delete','Agent Disconnect','Job Build','Job Cancel','Job Configure','Job Create','Job Delete','Job Discover','Job Move','Job Read','Job Workspace','Run Delete','Run Update','Run Replay','View Configure','View Create','View Delete','View Read','SCM Tag'],
+        'svinasi': ['Overall Read','Agent Build','Agent Configure','Agent Connect','Agent Create','Agent Delete','Agent Disconnect','Job Build','Job Cancel','Job Configure','Job Create','Job Delete','Job Discover','Job Move','Job Read','Job Workspace','Run Delete','Run Update','Run Replay','View Configure','View Create','View Delete','View Read','SCM Tag'],
```

Comment:
We do not favour granting individual permissions to employees unless absolutely necessary. Please add `svinasi` to the `pmm-devs` group on github. This is much faster and and does not require a PR.